dwave/system/samplers/leap_hybrid_sampler.py

```diff
@@ -918,7 +918,7 @@ def parameters(self) -> Dict[str, List[str]]:
 
     class SampleResult(NamedTuple):
         model: dwave.optimization.Model
-        timing: dict
+        info: dict
```

Comment:
I suppose we could say
```python
    info: dict[str, typing.Any]
```
might be overkill but :shrug: 

Even more overkill would be to use `typing.Literal`.

Reply:
And how about [this](https://github.com/dwavesystems/dwave-system/pull/565/commits/739a432c305d6fde248acc602746bf8d697cf314)? :smiley:

Reply:
So annotated!